tests/blockchain/eth/contracts/integration/test_integration.py

```diff
@@ -92,10 +93,12 @@ def policy_manager(testerchain, escrow):
 def user_escrow_proxy(testerchain, token, escrow, policy_manager):
     escrow, _ = escrow
     policy_manager, _ = policy_manager
+    secret_hash = testerchain.interface.w3.sha3(user_escrow_secret)
     # Creator deploys the user escrow proxy
     user_escrow_proxy, _ = testerchain.interface.deploy_contract(
         'UserEscrowProxy', token.address, escrow.address, policy_manager.address)
-    linker, _ = testerchain.interface.deploy_contract('UserEscrowLibraryLinker', user_escrow_proxy.address)
+    linker, _ = testerchain.interface.deploy_contract(
+        'UserEscrowLibraryLinker', user_escrow_proxy.address, secret_hash)
```

Comment:
OK - here is where we set the secret, right - Am I allowed to use *any* bytes as the secret?

Reply:
Yes, any bytes